sites/friday/src/get-started/django/deploy/configure.md

```diff
@@ -0,0 +1,107 @@
+- Give the project a name, something like `{{% vendor/cli %}}_django`.
+- Choose any combination of services you need for your project. If you are working with the project skeleton (selected the **Start from Scratch**) option, a common choice is **PostgreSQL** and **Redis**.
+
+This command will generate two files for you:
+
+- A `.environment` file, where you can save environment variables that will be sourced into the environment.
+- A `{{% vendor/configfile "app" %}}` file - the main {{% vendor/name %}} configuration file.
+
+1. **(Optional) Configure root**. The configuration generated assumes that the Django project root is the repository root.
+If this is not the case, and your Django code is nested within a directory, add the following `source.root` configuration to `{{% vendor/configfile "app" %}}`.
+
+    ```yaml {configFile="app"}
+    applications:
+        APP_NAME:
+            # ...GENERATED CONFIGURATION...
+
+            source:
+                root: NESTED_DIR
+
+            # ...GENERATED CONFIGURATION...
+    ```
+
+1. **Configure the build and deploy**. The `{{% vendor/configfile "app" %}}` contains a `hooks` block, containing configuration for how Django should be built and deployed. 
+The configuration for the "From scratch" app is shown below.
+Update your own build and deploy steps using it as an example.
+
+    ```yaml {configFile="app"}
+    applications:
+        APP_NAME:
+            # ...GENERATED CONFIGURATION...
+
+            hooks:
+                build: |
+                    set -eux
+
+                    # Upgrade pip
+                    python -m pip install --upgrade pip
+                    
+                    # Install dependencies
+                    pip install -r requirements/local.txt
+                    pip install -r requirements/production.txt
+
+                    # Collect static assets
+                    python manage.py collectstatic
+
+                deploy: |
+                    set -eux
+
+                    # Migrations.
+                    python manage.py migrate
+                    
+            # ...GENERATED CONFIGURATION...
+    ```
+
+1. **Configure how the app is started and served**. There is a `web` block in `{{% vendor/configfile "app" %}}` that describes a `start` command for Django, as well as other details for important path `locations` that should be served in specific ways. 
+The configuration for the "From scratch" app is shown below.
+Update your own build and deploy steps using it as an example.
+
+    ```yaml {configFile="app"}
+    applications:
+        APP_NAME:
+            # ...GENERATED CONFIGURATION...
+
+            web:
+                commands:
+                    start: "gunicorn -w 4 -b unix:$SOCKET APP_NAME.wsgi:application"
+
+                upstream:
+                    socket_family: unix
+
+                locations:
+                    "/":
+                        passthru: true
+                    "/static":
```

Comment:
If the user attempts to copy & paste the location `/static` that wasn't generated in `project:init`, when they paste into their config file, the indentation is off enough that it will cause parsing errors. 
<img width="344" alt="image" src="https://github.com/platformsh/platformsh-docs/assets/2332956/80bda9a3-b52f-4df2-a835-e2ac2f17c485">